### PR TITLE
python3-chromecast: update to 8.0.0.

### PR DESCRIPTION
**Pull request by uw2021**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by uw2021** [comment]
Just to explain the extra dependencies, it initially complained of what I assumed were missing dependencies (python3-pip and python3-wheel). Adding these allowed it to compile and install fine.

File: srcpkgs/python3-chromecast/template [hunk 1]
```diff
@@ -1,17 +1,17 @@
 # Template file for 'python3-chromecast'
 pkgname=python3-chromecast
-version=7.7.2
+version=8.0.0
 revision=1
 wrksrc="pychromecast-${version}"
 build_style=python3-module
-hostmakedepends="python3-setuptools"
+hostmakedepends="python3-setuptools python3-wheel python3-pip"
```
Comment (ericonr) [inline srcpkgs/python3-chromecast/template:7]: No, this is not how you fix the package. You need to add things from https://github.com/home-assistant-libs/pychromecast/blob/master/requirements.txt (they are in depends) to hostmakedepends.

Also add https://github.com/home-assistant-libs/pychromecast/blob/master/requirements-test.txt to checkdepends, if we have packages for all of them.

Comment (uw2021) [reply srcpkgs/python3-chromecast/template]: > No, this is not how you fix the package. You need to add things from https://github.com/home-assistant-libs/pychromecast/blob/master/requirements.txt (they are in depends) to hostmakedepends.
> 
> Also add https://github.com/home-assistant-libs/pychromecast/blob/master/requirements-test.txt to checkdepends, if we have packages for all of them.

Done. Thank you.

**Comment by ericonr** [comment]
Have you tested `mkchromecast` with this?

**Comment by uw2021** [comment]
> Have you tested `mkchromecast` with this?

I've just tested mkchromecast with both 8.0.0 and 7.7.2 and I receive the same error when running mkchromecast in the terminal:

`ImportError: cannot import name 'reboot' from 'pychromecast.dial' (/usr/lib/python3.9/site-packages/pychromecast/dial.py)

Also, when running mkchromecast from the app tray (on both 7.7.2 and 8.0.0) nothing happens. Not sure what's going on here.

**Comment by ericonr** [comment]
If it hits an ImportError it won't be able to launch at all, so it makes sense you didn't see anything.

**Comment by uw2021** [comment]
> If it hits an ImportError it won't be able to launch at all, so it makes sense you didn't see anything.

Is that related to my system? Strange it happens on both 7.7.2 and 8.0.0.

**Comment by ericonr** [comment]
Hm, we need to backport this PR https://github.com/muammar/mkchromecast/pull/325 , at least. They seem to have made a release for debian, I will try to find the tag being used.

**Comment by ericonr** [comment]
Alternatively, do what the debian package is doing and grab a git hash (I don't like it, but it works)

**Comment by uw2021** [comment]
> Hm, we need to backport this PR [muammar/mkchromecast#325](https://github.com/muammar/mkchromecast/pull/325) , at least. They seem to have made a release for debian, I will try to find the tag being used. [muammar/mkchromecast#348](https://github.com/muammar/mkchromecast/issues/348)

So if I've got this right, the plan is to try and get a 0.3.9 release, package it, then mkchromecast should work? Is using a git hash a better idea? Not familiar with it  to be honest.

**Comment by ericonr** [comment]
Better to use a release, but if upstream is unable to make one we should switch to a hash.

**Pull request merged**